transformer_engine/pytorch/module/layernorm_linear.py

```diff
@@ -411,7 +409,7 @@ def backward(
             )
 
             if ctx.cpu_offloading and ctx.fuse_wgrad_accumulation:
-                weight = torch.nn.Parameter(weight, False)
+                weight = torch.nn.Parameter(weight)
```

Comment:
```suggestion
                weight = torch.nn.Parameter(weight.requires_grad)
```

Reply:
Isn't it best in this case to simply retain the value.

Reply:
Hmm, CPU offloading works only for training. So, it is expected to have requires_grad=True. So, wanted to make it explicit and clear.

Reply:
If CPU offloading is enabled, then we should check for `weight.requires_grad` being true before instead of explicitly setting it here even if it was previously disabled (as there are use cases for this.)

Reply:
@sanandaraj5597 Any update here?